webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5072,6 +5076,7 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
                     (conversationAssigneeDetails.roleType == KommunicateConstants.APPLOZIC_USER_ROLE_TYPE.BOT) ? mckGroupLayout.checkBotDetail(conversationAssignee) : (CURRENT_GROUP_DATA.CHAR_CHECK = false);
                     
                 } else {
+                    params.isWaitingQueue=true;
```

Comment:
@Kusum-Dhoundiyal Can you check one case here?
- When socket connection is slow, were fetching conversation messages from message list API.
In that case, "viaCreateGroup" value will be true. So isWaitingQueue value should be true in that case also I suppose.